_includes/meta.html

```diff
@@ -4,91 +4,60 @@
 
 <!-- Twitter Card -->
 {% if page.twitter_card == "large" %}
```

Comment:
Most of the changes in this file are
- indentation
- moving duplicate tags outside of branches, using branches to assign variables (they all have `else` blocks so no chance of missing variables)